build/assets.js

```diff
@@ -1,65 +1,60 @@
+/* eslint-disable no-console */
 const fs = require('fs');
 const zlib = require('zlib');
-const Promise = require('bluebird');
-const klawSync = require('klaw-sync');
 const filesize = require('filesize');
 const Table = require('cli-table');
+const path = require('path');
+const sh = require('shelljs');
 
-const files = klawSync('dist/', {
-  ignore: ['examples', 'lang', 'font', '*.zip', '*.gz'],
-  nodir: true
-});
-
-Promise.all(files.map(gzipAndStat))
-.then(mapFiles)
-.then(function(files) {
-  logTable(files);
+const filepaths = sh.find(path.join(__dirname, '..', 'dist', '**', '*.{js,css}')).filter(function(filepath) {
+  if ((/\/(lang|example|font)\//).test(filepath)) {
+    return false;
+  }
 
-  return files;
-})
-.then(cleanup)
-.catch(function(err) {
-  console.error(err.stack);
+  return true;
 });
 
-function cleanup(files) {
-  files.forEach(function(file) {
-    fs.unlinkSync('dist/' + file[0] + '.gz');
-  });
-}
-
-function mapFiles(files) {
-  return files.map(function(file) {
-    const path = file[0].path;
-    const fileStat = file[0].stats;
-    const gzStat = file[1];
-    return [file[0].path.split('dist/')[1], filesize(fileStat.size), filesize(gzStat.size)];
-  });
-}
-
-function gzipAndStat(file) {
+Promise.all(filepaths.map(function(filepath) {
+  // gzip and stat
   return new Promise(function(resolve, reject) {
-    const readStream = fs.createReadStream(file.path);
-    const writeStream = fs.createWriteStream(file.path + '.gz');
+    const readStream = fs.createReadStream(filepath);
+    const writeStream = fs.createWriteStream(filepath + '.gz');
     const gzip = zlib.createGzip();
+
     readStream.pipe(gzip).pipe(writeStream).on('close', function() {
-      const gzStat = fs.statSync(file.path + '.gz');
+      const gzStat = fs.statSync(filepath + '.gz');
+      const fileStat = fs.statSync(filepath);
 
-      resolve([file, gzStat]);
+      resolve({filepath, gzStat, fileStat});
     })
-    .on('error', reject);
+      .on('error', reject);
   });
-}
-
-function logTable(files) {
```

Comment:
keeping these as named functions rather than inlining them into the promise chain increases readability of this file and the promise chain.